### PR TITLE
Revise analysis example for current API

### PR DESCRIPTION
**Pull request by rocky**

### CODE REVIEW

File: example/analysis [hunk 1]
```diff
@@ -1,48 +1,44 @@
 #!/usr/bin/env node
 
 "use strict"
+var fs = require('fs')
```
Comment (fgimenez) [inline example/analysis]: This should be a `const`

File: example/analysis [hunk 2]
```diff
@@ -1,48 +1,44 @@
...
-if (process.argv.length > 3) {
-    timeout = parseInt(process.argv[3])
-}
+const jsonPath = process.argv[2] || `${__dirname}/sample-json/PublicArray.json`
 
 /**********************************
   Example code starts here.....
 ***********************************/
 
+var options = {
```
Comment (fgimenez) [inline example/analysis]: `const` here too

File: example/analysis [hunk 3]
```diff
@@ -1,48 +1,44 @@
...
+    platforms: []  // client chargeback
+}
+
 const armlet = require('../index') // if not installed
 // const armlet = require('armlet') // if installed
 
 const client = new armlet.Client(options)
 
-client.analyze({ bytecode: bytecode, timeout: timeout }) // parameter "timeout" is optional
+client.analyze(options) // parameter "timeout" is optional
```
Comment (fgimenez) [inline example/analysis]: The comment about the timeout param is stale now.

**Review by fgimenez — DISMISSED** [review]
LGTM, a few small nitpicks.

**Comment by rocky** [comment]
Thanks for the suggestions. I think I have things covered now. But I make lots of mistakes so please double check. 

A little context on why this now - it is useful to have an up-to-date example to show Juan Blanco on how to use the API as it currently is, so we might be able to work on his VSCode solidity plugin.

P.S. 

At long last I don't feel so drawn out, that I have time to actually make changes suggested.

**Review by fgimenez — APPROVED** [review]
LGTM

**Pull request merged**